ax3_email/tasks.py

```diff
@@ -21,7 +22,7 @@ def _async_send_messages(serializable_email_messages, retries=RETRIES):
             if sent is not None:
                 count += 1
                 logger.info('Email sent to %s', message.to)
-        except (SMTPException, SMTPServerDisconnected) as exc:
+        except (SMTPException, SMTPServerDisconnected, RequestException) as exc:
```

Comment:
This handle all requests exceptions
```suggestion
        except (requests.exceptions.RequestException) as exc:
```